docs/_basic/listening_actions.md

```diff
@@ -6,7 +6,7 @@ order: 5
 ---
 
 <div class="section-content">
-Your app can listen to user actions like button clicks, menu selects, and message actions using the `action` method.
```

Comment:
@seratch maybe Japanese docs need this change too

Reply:
https://github.com/stevengill/bolt/pull/1

Reply:
This is already done.